web/ui/src/lib/clustersMap/campus/angouleme.ts

```diff
@@ -0,0 +1,63 @@
+import { Campus } from '../campus';
+import { Cluster } from '../cluster';
+import { ICampus } from '../types';
+
+//
+export class Angouleme extends Campus implements ICampus {
+  emoji = (): string => '🇫🇷';
+
+  name = (): string => 'Angouleme';
+
+  extractorRegexp = (): RegExp =>
+    /(?<clusterWithLetter>c(?<cluster>\d+))(?<rowWithLetter>r(?<row>\d+))(?<workspaceWithLetter>p(?<workspace>\d+))/i;
+
+  clusters(): Cluster[] {
+    return [
+      new Cluster({
+        identifier: 'c1',
+        name : "The silent one",
+        map: [
+          ['T:A', 'W:1A1', 'W:1A2', 'W:1A3', 'W:1A4', 'W:1A5', 'W:1A6', 'W:1A7', 'T:A'],
+          ['T:B', 'W:1B1', 'W:1B2', 'W:1B3',  null,   'W:1B4', 'W:1B5', 'W:1B6', 'T:B'],
+          ['T:C', 'W:1C1', 'W:1C2', 'W:1C3', 'W:1C4', 'W:1C5', 'W:1C6', 'W:1C7', 'T:C'],
+          ['T:D', 'W:1D1', 'W:1D2', 'W:1D3',  null,   'W:1D4', 'W:1D5', 'W:1D6', 'T:D'],
+          ['T:E', 'W:1E1', 'W:1E2', 'W:1E3', 'W:1E4', 'W:1E5', 'W:1E6', 'W:1E7', 'T:E'],
+          ['T:F', 'W:1F1', 'W:1F2', 'W:1F3',  null,   'W:1F4', 'W:1F5', 'W:1F6', 'T:F'],
+          ['T:G', 'W:1G1', 'W:1G2', 'W:1G3', 'W:1G4', 'W:1G5', 'W:1G6', 'W:1G7', 'T:G'],
+          ['T:H', 'W:1H1', 'W:1H2', 'W:1H3', 'W:1H4', 'W:1H5', 'W:1H6', 'W:1H7', 'T:H'],
+        ],
+      }),
+      new Cluster({
+        identifier: 'c2',
```

Comment:
The identifier cannot be `c2` has your location haven't `c` in location, on your campus your identifier start with the number of cluster directly

```suggestion
        identifier: '2',
```